packages/react-components/react-avatar/src/components/AvatarGroupItem/useAvatarGroupItem.ts

```diff
@@ -60,6 +60,7 @@ export const useAvatarGroupItem_unstable = (
     overflowLabel: resolveShorthand(props.overflowLabel, {
       required: true,
       defaultProps: {
+        'aria-hidden': true,
```

Comment:
Could you add a comment about why this is needed? I.e. that the Avatar itself already has its aria-label set to the name, so we want to hide this to prevent the name from being read twice.